js/templates/modals/settings/addressesForm.html

```diff
@@ -0,0 +1,83 @@
+<form class="box border padKids padStack clrS clrBr js-addAddressForm">
+  <div class="flexRow">
+    <div class="col3">
+      <label for="settingsAddressName"><%= ob.polyT('settings.addresses.name') %></label>
+    </div>
+    <div class="col9">
+      <% if (ob.errors.name) print(ob.formErrorTmpl({ errors: ob.errors.name })) %>
+      <input type="text" class="clrBr" name="name" for="settingsAddressName" value="<%= ob.name %>" />
```

Comment:
Why do the inputs have for attributes? That's not a valid attribute for an Input element, should it be an Id?